### DISCUSSION TITLE
Add a little reminder to remind players that you have to hold to delete (or discard)

### DISCUSSION THREAD

**Discussion by MeguMario** [ppy/osu#17808] [Ideas]
Due to my unawareness of the change to the discard & delete button behavior, I've created this report https://github.com/ppy/osu/issues/17806 under the assumption that this was a bug. Now that I know that you have to hold (and button will also have a tracking effect on it, thanks guys), I think that it's not clear enough (at least for someone who just return to lazer or an idiot like me) to just feature a tracking effect on the button telling people that you have to hold it, a text reminder would be fine, something like "Please hold the button to proceed" after a player has attempted to click on it multiple time.

**Comment by bdach** [comment]
do you have this problem with the "exit gameplay" button? or the retry hotkeys? those also use animations to convey that you need to hold rather than click. it's a bit of an established UX pattern.

**Reply by MeguMario** [reply]
the retry hotkey works at it should like it's on stable so i don't have problem with it, the exit gameplay button, at least if it was this button here, then i think it has text to tell me that you have to hold it (also the animation are much more prominent on a black background, but then again that could just be i consciously noticed it now, so i'm not sure), so not much problem either
![image](https://user-images.githubusercontent.com/44724359/163256471-b44c7bd0-3605-469c-b3ff-bb6079689106.png)

**Reply by MeguMario** [reply]
tho now that you mention, i don't know if the retry hotkey was mentioned its existence in game outside of the keybind section. Maybe having it in the startup game tips would be nice (or an indicator next to the retry button ingame). Just for those who didn't know that you can do that.

**Reply by integerrr** [reply]
the problem i have with beatmap delete hold animation is the 2 colours not being contrasted well enough (red and a lighter shade of red), it doesnt stand out enough to me, personally the first time i deleted beatmap, it took me a few clicks of the button and some ??? moments before i figured it out.
This is not an issue with the exit gameplay button as the progress bar (being white) contrasts very well with the black button.
